kotlin/private/kt_android_library_and_test.bzl

```diff
@@ -123,6 +126,7 @@ def kt_android_library_and_test(
     for test_class in instrumented_test_classes:
         kt_android_local_test(
             name = scope_name(name, "%s-instrumented-test" % test_class.split(".")[-1]),
+            manifest = manifest,
```

Comment:
Given that manifests can be defined specifically for instrumented tests, we should accept a specific `instrumented_manifest` that defaults to `None` (falling back to `manifest`).

This would mean `kt_android` would need to take in `instrumented_manifest` and default to `src/androidTest/AndroidManifest.xml`, if found.